pkg/plugins/trivy/plugin.go

```diff
@@ -605,6 +626,30 @@ func (p *plugin) getPodSpecForStandaloneMode(ctx trivyoperator.PluginContext, co
 			SubPath:   ".trivyignore",
 		})
 	}
+	if config.FindIgnorePolicyKey(workload) != "" {
+		volumes = append(volumes, corev1.Volume{
+			Name: ignorePolicyVolumeName,
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: trivyConfigName,
+					},
+					Items: []corev1.KeyToPath{
+						{
+							Key:  config.FindIgnorePolicyKey(workload),
+							Path: "policy.rego",
+						},
+					},
+				},
+			},
+		})
+
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{
+			Name:      ignorePolicyVolumeName,
+			MountPath: "/etc/trivy/policy.rego",
+			SubPath:   "policy.rego",
+		})
+	}
```

Comment:
maybe this code be exported to function and reused in client server mode and other

Reply:
@chen-keinan Done. I also did this refactor for the volumes of the .trivyignore file. In that I noticed, that there were two cases:

1. In two cases it was mounted to `/etc/trivy/.trivyignore` (I assume this is the intended location)
2. In one case it was mounted to `/tmp/trivy/.trivyignore` (I assume this was just a typo)

Now the files always go to `/etc/trivy/.trivyignore` (`/etc/trivy/policy.rego` respectively)